examples/em-jsontemplates/GeneralPerfExperimentsTemplate.json

```diff
@@ -1,212 +1,222 @@
 [
-  {
-    "experiment_name": "quarkus-resteasy-autotune-min-http-response-time-db",
-    "mode": "Apply/MetricCollectionOnly",
-    "environment": "QA/PROD",
-    "resource": {
-      "namespace": "default",
-      "deployment_name": "tfb-qrh-sample"
-    },
-    "datasource_info": {
-      "prometheus1": {
-        "url": "http://10.101.144.137:9090",
-        "provider": "prometheus"
-      }
-    },
-    "trial_info": {
-      "trial_id": "",
-      "trial_num": 0,
-      "trial_result_url": "http://localhost:8080/listExperiments?experiment_name=quarkus-resteasy-autotune-min-http-response-time-db"
-    },
-    "settings": {
-      "trial_settings": {
-        "measurement_cycles": "3",
-        "warmup_duration": "1min",
-        "warmup_cycles": "3",
-        "measurement_duration": "1min",
-        "iterations": "3"
+    {
+      "experiment_name": "quarkus-resteasy-autotune-min-http-response-time-db",
+      "resource": {
+        "namespace": "default",
+        "deployment_name": "tfb-qrh-sample"
       },
-      "deployment_settings": {
-        "deployment_policy": {
-          "type": "rollingUpdate"
+      "datasource_info": {
+        "prometheus1": {
+          "url": "http://10.101.144.137:9090",
+          "provider": "prometheus"
         }
-      }
-    },
-    "pod_metrics": {
-      "request_sum": {
-        "datasource": "prometheus1",
-        "query": "rate(http_server_requests_seconds_sum{method=\"GET\",outcome=\"SUCCESS\",status=\"200\",uri=\"/db\",}[1m])",
-        "name": "request_sum"
       },
-      "request_count": {
-        "datasource": "prometheus1",
-        "query": "rate(http_server_requests_seconds_count{method=\"GET\",outcome=\"SUCCESS\",status=\"200\",uri=\"/db\",}[1m])",
-        "name": "request_count"
-      }
-    },
-    "container_metrics": {
-      "container_name": {
-        "memoryRequest": {
-          "name": "memoryRequest",
-          "query": "container_memory_working_set_bytes{$CONTAINER_LABEL$=\"\", $POD_LABEL$=\"$POD$\"}",
-          "datasource": "prometheus",
-          "valueType": "double"
+      "trial_info": {
+        "trial_id": "",
+        "trial_num": 0,
+        "trial_result_url": "http://localhost:8080/listExperiments?experiment_name=quarkus-resteasy-autotune-min-http-response-time-db"
+      },
```

Comment:
`trial_info` is appearing redundant now. Since `trial_num` is already part of the `trials` section. So I suggest you add the `trial_result_url` for each individual trial in the `trials` section, remove `trial_id` as we have never used it so far. This way we can completely eliminate the `trial_info` section